Firmware/ultralcd.cpp

```diff
@@ -3717,8 +3717,7 @@ static void lcd_wizard_load() {
         lcd_puts_at_P(0, 2, _T(MSG_LOADING_FILAMENT));
         loading_flag = true;
     }
-    gcode_M701(FILAMENTCHANGE_FIRSTFEED, 0);
```

Comment:
Note I hope this change works OK. Otherwise we need another call to `raise_z_above(MIN_Z_FOR_LOAD)` here.

Reply:
This seems to differ. "M701" will call gcode_M701(FILAMENTCHANGE_FIRSTFEED, 0xff) which seems to bypass ``MMU2::mmu2.load_filament_to_nozzle(mmuSlotIndex);``. Here we force slot 1 instead.
I can't test this at the moment to verify...

Reply:
You're correct!

I also noticed if slot `0xFF` is given with the MMU, then the M701 will act like the single color setup which is not good. I should probably split that MMU if statement into two nested if's even though it may take 4-8 bytes.
![image](https://user-images.githubusercontent.com/8218499/235798367-ca35b5a0-dc20-47f8-baf6-8af72510a7e0.png)

Reply:
Should hopefully be fixed now. Needs to be verified.

Reply:
Looks good now.

It's almost a shame we have to put the if for all those raise_z_above().

During M701/M702 the Z parameter is always relative to the current position?

Reply:
> During M701/M702 the Z parameter is always relative to the current position?

Yes I believe that's correct. Because we use `raise_z` for the lift, which applies a delta to whatever the current position is. The delta is then also used to revert the Z move similar to how we unpark the XY axis in M600.